app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutUploader.java

```diff
@@ -253,6 +267,7 @@ private void populateV1APICalibrationEntry(JSONArray array, Calibration record)
                 json.put("intercept", (long) ((record.intercept * -1000) / (record.slope * 1000)));
                 json.put("scale", 1);
             }
+            json.put("sysTime", format.format(record.timestamp));
```

Comment:
yes, this was discussed with Jason on gitter on first of december.

The main reason that this is need is since this the field that NS are using to not have duplicates.
If this is not used, then for each calibration/meter record we will have two records representing the one measurement that we have done.